includes/class-user-manual-sync.php

```diff
@@ -0,0 +1,239 @@
+<?php
+/**
+ * Manually sync individual users to network sites.
+ *
+ * @package Newspack
+ */
+
+namespace Newspack_Network;
+
+use Newspack\Data_Events;
+use Newspack_Network\Debugger;
+
+/**
+ * Class to watch the user for updates and trigger events
+ */
+class User_Manual_Sync {
+
+	/**
+	 * The metadata we're syncing.
+	 *
+	 * @var array
+	 */
+	public static $meta_to_sync = [
```

Comment:
Let's not duplicate it, but read from that class instead

Reply:
That totally makes sense! I updated the `$user_props` as well to grab from the same place.